types/draft-js/index.d.ts

```diff
@@ -151,10 +151,9 @@ declare namespace Draft {
                  * that you set this to `true`.
                  */
                 stripPastedStyles?: boolean | undefined;
-                formatPastedText?: ((
-                    text: string,
-                    html?: string,
-                ) => { text: string, html: string | undefined }) | undefined,
+                formatPastedText?:
+                    | ((text: string, html?: string) => { text: string; html: string | undefined })
+                    | undefined;
```

Comment:
I guess this is okay, but I get different results when using `DefinitelyTyped`'s Prettier config 🤔 

How did you format this, sorry @fondation451?

This is what I get:
```suggestion
                formatPastedText?: (text: string, html?: string) => { text: string; html: string | undefined };
```

Reply:
@PeterDekkers I am running this command `yarn prettier types/draft-js/index.d.ts --write` which should use the prettier config of the project.
Is there another way to run it for the repository?